src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Microsoft.Plugin.Folder.csproj

```diff
@@ -134,6 +134,12 @@
     <None Update="Images\user.light.png">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Images\Warning.dark.png">

```

Comment:
These images need to be added to the installer.

Reply:
Like